tests/usbredir_test.go

```diff
@@ -65,9 +66,8 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 
 		var vmi *v1.VirtualMachineInstance
 		BeforeEach(func() {
-			vmi, err = createVMI(virtClient, false)
-			Expect(err).ToNot(HaveOccurred())
-			tests.WaitForSuccessfulVMIStart(vmi)
+			vmi = libvmi.New(libvmi.WithResourceMemory("32Mi"))
```

Comment:
What do you think about making this 32Mi a named constant. A constant that we can also comment.

Another person will come here, and try to improve, and will be asking why 32 Mi and not 1Mi or 128Mi?

Reply:
We had a discussion about the 32Mi:
https://github.com/kubevirt/kubevirt/pull/8075#discussion_r919005138

I definitely  agree that we should a follow up PR for `libvmi.NewDIskLess()` with the right default .
I plan to work on it soon.
But in my opinion this is out of the scope of this PR .

Reply:
The thing is a well named constant is self-explanatory. 

Please consider the amount of digging required to find that comment when you're browsing the code (at least git blame to find out which commit - then find out which PR - then browse PR (hopefully the comment is not resolved ...). It's magnitudes harder than reading what it is from a const.

... lazy people will just roll with it. 

I would encourage you to make the reader's life easier.

Reply:
That's a good point i added a constant for now.

Reply:
now it looks better